mobify/pinny
# Adding destroy call for pinny

Added destroy function.

Status: **Ready to merge**

Reviewers: @kpeatt @tedtate 
JIRA: https://mobify.atlassian.net/browse/CSOPS-948
Linked PRs: https://github.com/mobify/shade/pull/11

## Changes
- added `destroy` function
- updated tests with `destroy` function

## How to test-drive this PR
- checkout this branch
- `grunt test`
- `grunt serve` and check out the example page
- in the console, call `destroy` on a pinny and ensure it's destroyed
